enos/enos-scenario-e2e-static-with-vault.hcl

```diff
@@ -16,6 +16,7 @@ scenario "e2e_static_with_vault" {
   locals {
     aws_ssh_private_key_path = abspath(var.aws_ssh_private_key_path)
     boundary_install_dir     = abspath(var.boundary_install_dir)
+    boundary_license_path    = abspath(var.boundary_license_path != null ? var.boundary_license_path : joinpath(path.root, "./support/boundary.hclic"))
```

Comment:
I was thinking about the `joinpath(path.root, "./support/boundary.hclic")` part here and was wondering if that was the appropriate thing to set here. I haven't thought through alternatives, though...

Reply:
it's a default we use in vault for CI, but up for changes!

Reply:
We can just leave it like this for now.